docs/troubleshooting.rst

```diff
@@ -202,7 +202,7 @@ thread then call ``allow_sta()`` before calling any other Bleak APis::
         pass
 
```

Comment:
Should mention setting `sys.????` as workaround to prevent pywin32 from initializing the thread in the first place.